lib/src/index.ts

```diff
@@ -199,4 +199,13 @@ export { default as NxCodeSnippet, Props as NxCodeSnippetProps } from './compone
 
 export { default as useToggle } from './util/useToggle';
 
+export { default as NxNavigationSidebar, Props as NxNavigationSidebarProps }
+  from './components/NxGlobalSidebar/NxGlobalSidebar';
```

Comment:
Shouldn't we export as `NxGlobalSidebar` and `NxGlobalSidebarProps`

Reply:
> Shouldn't we export as `NxGlobalSidebar` and `NxGlobalSidebarProps`

Yup. Good catch — a leftover from previous naming
Fixed in 31ac81b91131d8cc4b540415e282d57d06def6e4